tx/textual/internal/testdata/any.json

```
@@ -0,0 +1,72 @@
+[
+    {
+        "url": "type.googleapis.com/Foo",
+        "msg": {},
+        "screens": [
+            {"text": "Object: type.googleapis.com/Foo"},
+            {"text": "Foo object", "indent": 1}
+        ]
+    },
+    {
+        "url": "type.googleapis.com/Foo",
+        "msg": {
+            "full_name": "testing"
+        },
+        "screens": [
+            {"text": "Object: type.googleapis.com/Foo"},
+            {"text": "Foo object", "indent": 1},
+            {"text": "Full name: testing", "indent": 2}
+        ]
+    },
+    {
+        "url": "type.googleapis.com/google.protobuf.Timestamp",
+        "msg": {
+            "seconds": 1136214245
+        },
+        "screens": [
+            {"text": "Object: type.googleapis.com/google.protobuf.Timestamp"},
+            {"text": "2006-01-02T15:04:05Z", "indent": 1}
+        ]
+    },
+    {
+        "url": "type.googleapis.com/google.protobuf.Any",
+        "msg": {
+            "type_url": "type.googleapis.com/Foo",
+            "value": []
+        },
+        "screens": [
+            {"text": "Object: type.googleapis.com/google.protobuf.Any"},
+            {"text": "Object: type.googleapis.com/Foo", "indent": 1},
+            {"text": "Foo object", "indent": 2}
+        ]
+    },
+    {
+        "url": "type.googleapis.com/google.protobuf.Any",
+        "msg": {
+            "type_url": "type.googleapis.com/Foo",
+            "value": [10, 7, 116, 101, 115, 116, 105, 110, 103]
```

Same, if we go protojson as above, then this 2 lines can be replaced by

```suggestion
            "@type": "type.googleapis.com/Foo",
            "full_name": "testing"
```

which I think it more desirable, since these tests will also be read by humans

---

Done. Doesn't allow me to show off my mental encoding skills, but I get plenty of that in the CBOR PR.